blueprints/empress-blog/files/__base__/page/chris-manson.md

```diff
@@ -9,6 +9,10 @@ authors:
   - ghost
 date: Tue Jun 12 2018 17:50:55 GMT+0100 (IST)
 tags:
+meta:
```

Comment:
same suggestion as above: move this to the dummy app rather than the blueprint

Reply:
Same answer as above.